reaper/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "reaper",
+  "version": "1.0.0",
+  "description": "Runs the reaper lambda code",
+  "main": "lambda.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "riffraff-artefact": "VERBOSE=true ARTEFACT_PATH=$PWD riffraff-artefact"
+  },
+  "isAwsLambda": true,
+  "cloudformation": false,
+  "projectName": "media-service::grid::reaper-lambda",
+  "uploadArtefact": true,
+  "riffraffFile": "./riff-raff.yaml",
+  "author": "",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@guardian/elk-kinesis-logger": "0.1.0",
+    "aws-sdk": "^2.127.0",
+    "node-fetch": "^1.7.3",
+    "node-riffraff-artefact": "^2.0.1"
```

Comment:
I think `aws-sdk` and `node-riffraff-artefact` should be dev dependencies. `aws-sdk` is included in the lambda environment by default.